### ISSUE TITLE
prompt blocks the terminal when use in promise chain 

### ISSUE THREAD

**Issue by ccamensuli** [SBoudrias/Inquirer.js#811]
in this new version 6.4.0 
node v12.4.0

when i use in promise chain the second call to prompt method  block the terminal  

>> test.js

#!/usr/bin/env node

const inquirer = require("inquirer");

const prompt = inquirer.createPromptModule();

prompt(questions)
  .then((answers) => {
    console.log('\nOrder receipt:');
    return JSON.stringify(answers);
  })
  .then((lastresponse) => {
    return prompt(questions)
      .then(function(answers) {
        console.log('\nOrder receipt 2:');
        console.log(lastresponse);
        console.log(JSON.stringify(answers, null, '  '));
        return answers;
      });
  })
  .catch(e => {
    console.error(e);
  });

const questions = [{
    type: 'confirm',
    name: 'toBeDelivered',
    message: 'Is this for delivery?',
    default: false
  }, {
    type: 'input',
    name: 'phone',
    message: 'What\'s your phone number?',
    validate: function(value) {
      var pass = value.match(/^([01]{1})?[-.\s]?\(?(\d{3})\)?[-.\s]?(\d{3})[-.\s]?(\d{4})\s?((?:#|ext\.?\s?|x\.?\s?){1}(?:\d+)?)?$/i);
      if (pass) {
        return true;
      }
      return 'Please enter a valid phone number';
    }
  }, {
    type: 'list',
    name: 'size',
    message: 'What size do you need?',
    choices: ['Large', 'Medium', 'Small'],
    filter: function(val) {
      return val.toLowerCase();
    }
  }, {
    type: 'input',
    name: 'quantity',
    message: 'How many do you need?',
    validate: function(value) {
      var valid = !isNaN(parseFloat(value));
      return valid || 'Please enter a number';
    },
    filter: Number
  }, {
    type: 'expand',
    name: 'toppings',
    message: 'What about the toppings?',
    choices: [{
        key: 'p',
        name: 'Pepperoni and cheese',
        value: 'PepperoniCheese'
      }, {
        key: 'a',
        name: 'All dressed',
        value: 'alldressed'
      }, {
        key: 'w',
        name: 'Hawaiian',
        value: 'hawaiian'
      }
    ]
  }, {
    type: 'rawlist',
    name: 'beverage',
    message: 'You also get a free 2L beverage',
    choices: ['Pepsi', '7up', 'Coke']
  }, {
    type: 'input',
    name: 'comments',
    message: 'Any comments on your purchase experience?',
    default: 'Nope, all good!'
  }, {
    type: 'list',
    name: 'prize',
    message: 'For leaving a comment, you get a freebie',
    choices: ['cake', 'fries'],
    when: function(answers) {
      return answers.comments !== 'Nope, all good!';
    }
  }
];

**Comment by rspieker** [comment]
I ran into the same issue. The easier step to reproduce is basically to invoke `inquirer.prompt` multiple times.

```
const inquirer = require("inquirer");

Promise.resolve()
  .then(() =>
    inquirer.prompt({ type: "confirm", name: "thing", message: "Do the thing", default: true })
  )
  .then(() =>
    inquirer.prompt({ type: "input", name: "sure", message: "Are you sure" })
  );
```
This will present you with two prompts, whereas the second one does not allow input (and appears to trap CTRL+C too)

Tested on node v8.16.0 and v10.15.3

**Comment by hoang-innomizetech** [comment]
Me too. We use this as a dependencies of pm2-windows-service

`npm install @innomizetech/pm2-windows-service
`
![image](https://user-images.githubusercontent.com/43955692/59840962-a03c1d00-937d-11e9-9813-421f6280748b.png)


We cannot type anything and also CTRL+C also not working.

**Comment by danielo515** [comment]
Yep the same is happening to me, only on the latest version.
Not sure what to do.

**Comment by hoang-innomizetech** [comment]
It seems working with v6.3.1, maybe windows updates cause this issue.

**Comment by rspieker** [comment]
Until this is fixed, one way to fix it is to make sure inquirer is bound to 6.3.x

I recon for most people this comes down to changing the dependency in `package.json`

from
```
"inquirer": "^6.3.1"
```
to 
```
"inquirer": "~6.3.1"
```

(`^` means the package will use the latest patch AND feature (minor) version, `~` means it will use the latest patch version and does not move over to a newer feature (minor) version)

**Comment by sodatea** [comment]
Caused by this commit: https://github.com/SBoudrias/Inquirer.js/pull/808

**Comment by chrisdothtml** [comment]
PR here: #813

**Comment by hongxuWei** [comment]
> It seems working with v6.3.1, maybe windows updates cause this issue.

No, I can't work it at 6.3.1 on MacOS

**Comment by SBoudrias** [comment]
I just released `6.4.1` with the event listeners revert.

Thanks all for your patience.
